### ISSUE TITLE
Error summary/message for missing answers on required fields (Update 1)

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#4502]
## Problem
Required questions provide ambiguous error messages when left empty. This violates WCAG criteria 3.3.3.
> Based on past issues #401, #424, #449, #3950, 

## Mitigation
Update content to be more specific:
- Mention question type (ex: text field)
- Mention action required (ex: text entry, selection)
- Add truncated question label at the end for additional context (ex: "Address"

## Content update
**Complete all required questions:**
- Answer the text field for [EN truncated text input type question label]
- Select an option for: [EN truncated radio button or dropdown type question label]
- Select one or multiple options for: [EN truncated checkbox type question label]
- Check off the boxes for: [EN trunctated attestation type question label]

## Additional detail
https://docs.google.com/spreadsheets/d/1qx6K-PkburkQ2anDbRAhLpgWIfbYF8aWV6N34NrFUp8/edit?gid=0#gid=0

**Comment by anikbrazeau** [comment]
@timarney Sure, that would be great!

**Comment by timarney** [comment]
Started a PR 

Nothing to show yet but looks like this is feasible to do :) 

https://github.com/cds-snc/platform-forms-client/pull/4934

**Comment by anikbrazeau** [comment]
# Work underway here: #4934 

<img width="881" alt="Screenshot 2025-01-10 at 8 49 43 AM" src="https://github.com/user-attachments/assets/b5ed4c6b-d61d-4a1d-a9c5-53095c44a181" />

<img width="875" alt="Screenshot 2025-01-10 at 8 48 09 AM" src="https://github.com/user-attachments/assets/16317730-92b2-4a50-b0c4-8107e38d225b" />

<img width="882" alt="Screenshot 2025-01-10 at 8 48 38 AM" src="https://github.com/user-attachments/assets/fb957908-8687-48dc-a27d-0be0ad2cc4e0" />

**Comment by timarney** [comment]
Merged.
